lib/openhab/dsl/items/rollershutter_item.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+require 'java'
+
+module OpenHAB
+  module DSL
+    module Items
+      #
+      # Delegator to OpenHAB Rollershutter Item
+      #
+      class RollershutterItem
```

Comment:
This should extend numeric?

Reply:
Not sure what the benefit would be? But wouldn't be a problem to add

Reply:
See below, it is what should make it automatically work in a case statement.